react/testing_2/src/screens/Dashboard/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+import logo from './logo.svg';
+import './styles.css';
+import { DATA } from './constants';
+import UserList from './components/UserList';
+
+class Dashboard extends Component {
+  state = { users: [] };
+  componentDidMount() {
+    this.setState({ users: DATA });
```

Comment:
avoid using setState in componentDidMount. We could do this in 2 ways:

1) in constructor, just assign state directly: `this.state = DATA`
2) Use a Promise to "simulate" API, call it in componentDidMount and in its callback use `setState`